### ISSUE TITLE
0 instances available

### ISSUE THREAD

**Issue by rlnovak** [ralph-group/mucloud#11]
Hello! I followed your instruction to setup up mucloud and AWS, but I'm getting the following error message:

Traceback (most recent call last):
  File "<string>", line 671, in <module>
  File "<string>", line 547, in launch_instance
  File "<string>", line 434, in launch
  File "boto/ec2/connection.py", line 973, in run_instances
  File "boto/connection.py", line 1207, in get_object
boto.exception.EC2ResponseError: EC2ResponseError: 400 Bad Request
<?xml version="1.0" encoding="UTF-8"?>
<Response><Errors><Error><Code>InstanceLimitExceeded</Code><Message>You have requested more instances (1) than your current instance limit of 0 allows for the specified instance type. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.</Message></Error></Errors><RequestID>df206ae3-9c8b-4bb5-b8fa-8007a9d8c982</RequestID></Response>
mucloud returned -1

Trying to launch from the AWS EC2 console also leads to the same error:
"You have requested more instances (1) than your current instance limit of 0 allows for the specified instance type. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit."

Please, what have I missed?

**Comment by cjermain** [comment]
@rlnovak, I have not encountered this before, but I'm happy to help you troubleshoot. What region are you launching your instances in? Do you have other instances running at the same time (perhaps of different types)?

**Comment by rlnovak** [comment]
I configured AWS for US East (N. Virginia), following the instructions.
I have no instances running. It is the first time I use Mucloud and AWS.

I checked the 'Limits' on my EC2 dashboard, and in fact AWS gave me 0 g2.2xlarge instances to run. I requested them to increase this, and now I am waiting for their answer. I think maybe the problem is with my AWS account, since you haven't encountered this issue before.

![screen shot 2017-03-20 at 2 54 18 pm](https://cloud.githubusercontent.com/assets/26547102/24114058/a37e17c6-0d7d-11e7-855d-475dc838c9a1.png)
